src/net/WebApp/Pages/Index.razor

```diff
@@ -12,15 +12,18 @@
         </div>
         <div class="col-12">
             <div class="form-row">
-                <div class="col col-6">
+                <div class="col col-2">
                     <EditForm OnValidSubmit="HandleAddFormSubmit" Model="@image">
                         <DataAnnotationsValidator />
                         <ValidationSummary />
                         <button type="submit" value="submit" disabled="@(!IsConnected)" class="btn shadow-none" style="cursor:pointer"><span style="font-size: 18px" class="oi oi-plus" title="Add Meme"
                                 aria-hidden="true"></span></button>
                     </EditForm>
                 </div>
-                <div class="col col-6">
+                <div class="col col-8">
+                    <input type="text" @bind-value="searchTerm"/>
+                </div>
+                <div class="col col-2">

```

Comment:
We aren't going to add this ability for now...if you want to hid you can do via AppConfiguration service setting.

Reply:
removed altogether